### PR TITLE
Update wireguard-client.rst to clarify use of WireGuard net

### PR DESCRIPTION
**Pull request by Greelan**
This proposed tweak to the how-to is intended to address the issue discussed in this forum thread: https://forum.opnsense.org/index.php?topic=21170.0

### CODE REVIEW

**Comment by fichtner** [comment]
@mimugmail ping :)

File: source/manual/how-tos/wireguard-client.rst [hunk 1]
```diff
@@ -93,11 +93,15 @@ The final piece is to allow traffic from the Wireguard network. Do this via :men
...
  **Interface**               WireGuard        *The interface this rule applies to*
- **Source**                  WireGuard net    *Source subnet*
+ **Source**                  WireGuard net    *Source subnet - but see the warning below*
  **Destination**             any              *Traffic destination*
  **Description**             WG WAN to LAN    *Optional - provide a description*
 =========================== ================ =====================================================================
 
+.. Warning::
+
+    Using **WireGuard net** as the Source will not work as expected if multiple local WireGuard peers or multiple WireGuard endpoints are configured. In those cases, define an Alias that includes the IP addresses (for example, 10.10.10.2 and 10.10.10.3) or subnet (for example, 10.10.10.0/24) of the VPN clients that are intended to have LAN access. Then include that Alias as the Source in the firewall rule.
```
Comment (mimugmail) [inline source/manual/how-tos/wireguard-client.rst:103]:
```suggestion
    Using **WireGuard net** as source will not work as expected if multiple local WireGuard instances or WireGuard endpoints are configured. In those cases, define an alias including the IP addresses (e.g. 10.10.10.2 and 10.10.10.3) or subnet (e.g. 10.10.10.0/24) of the VPN clients that are intended to have access.
```

Comment (mimugmail) [reply source/manual/how-tos/wireguard-client.rst:103]: I'm not a native speaker but would write it like this.

**Comment by Greelan** [comment]
These largely look like stylistic changes, which I am not going to get upset about (even though I am a native speaker 😀). Deleting the last sentence I included does however mean the instructions only explicitly tell the user to define an Alias, but not to use it in the rule. Implicit (and obvious), sure, but some people seem to need things spelled out.

File: source/manual/how-tos/wireguard-client.rst (hunk 1 above)
Comment (Greelan) [reply source/manual/how-tos/wireguard-client.rst:103]: These largely look like stylistic changes, which I am not going to get upset about (even though I am a native speaker 😀). Deleting the last sentence I included does however mean the instructions only explicitly tell the user to define an Alias, but not to use it in the rule. Implicit (and obvious), sure, but some people seem to need things spelled out.

**Comment by sovanyio** [comment]
Primarily here to say THANK YOU, I have spent more time than I care to admit debugging why I couldn't get wireguard to work. I now have a working setup :)

Second, I think that the note under 2c for `Source address` might still be a bit misleading to some folks.

It might be more clear to have the warning as its own section prior to sections b and c. Perhaps something like `Creating Wireguard Network Alias`. I think it will be far more common that someone sets up a multi-endpoint setup and that the single endpoint/single-server folks should be the exception in the instructions:

`Info: If using a single endpoint with a single server, it is possible to use "Wireguard Net" rather than create an alias.`

**Comment by Greelan** [comment]
Fair comment. I will have a look at some revised wording along those lines.

**Comment by Greelan** [comment]
@mimugmail Do you have any comments on the revised text? Thanks

**Comment by Greelan** [comment]
> Looks good, thank you :)

Thanks. Look forward to it finally being merged - nudge, @fichtner 😛

**Comment by fichtner** [comment]
Merged, thanks! :)  *goes back to sleeping on the desk*

**Pull request merged**